src/main/java/com/google/devtools/build/lib/rules/genrule/GenRuleBase.java

```diff
@@ -321,6 +326,31 @@ public String lookupVariable(String variableName) throws ExpansionException {
         return expandSingletonArtifact(filesToBuild, "$@", "output file");
       }
 
+      if (variableName.equals("OUT_ROOT_DIR")) {
```

Comment:
Please also add a test to `GenRuleCommandSubstitutionTest`.

Reply:
added, ptal

Reply:
Added a test in `GenRuleConfiguredTargetTest` - seemed like the appropriate spot. lmk if I should move etc